frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -137,8 +137,12 @@ export const CreateOCS = withHandlePromise<CreateOCSProps & HandlePromiseProps>(
   React.useEffect(() => {
     if ((pvLoadError || pvData.length === 0) && pvLoaded) {
       setFilteredNodes([]);
```

Comment:
One thing to point out is that we are not handling anywhere the fallback UX when we have no `filterednodes`.
It will show up the list page with table headers and no rows.
I am not sure why this is so, because if we get empty data in list page, we see "No Nodes Found" like message.
![Screenshot from 2020-09-28 14-12-22](https://user-images.githubusercontent.com/25664409/94410155-aa5de580-0194-11eb-9a17-894a216a8d48.png)

Reply:
Ack. This change needs to be done at multiple places and hence, I will take this change as a part of separate PR. 
This PR is urgent and needs to be get in asap.

Reply:
I am okay to see it in another PR as well.